utils/interfaces/_library/miscs.py

```diff
@@ -101,6 +102,10 @@ def check(self, data):
 
 
 class _TraceExistence(BaseValidation):
+    def __init__(self, request, span_type):
+        super().__init__(request=request)
+        self.span_type = span_type.lower()
```

Comment:
Remove `lower` (no reason to be too permissive), and it''' allow `None` value

Reply:
Done